doc/rules/basic/curly_braces_position.rst

```diff
@@ -19,7 +19,7 @@ Default value: ``'same_line'``
 ``functions_opening_brace``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-the position of the opening brace of functions body.
+The position of the opening brace of functions body.
```

Comment:
I believe it should be `function's` or `functions'`, regardless if singular/plural, it should be saxon genitive.

Similar improvement could be added in other places (like `control structures` above).

Reply:
I'm not sure about this one, is any of the 3 versions - the current one and your 2 suggestions - grammatically correct, and the others aren't?

Reply:
The current one is incorrect. Suggestions are both correct but it depends which form (singular/plural) is intended.

Reply:
> The current one is incorrect

It is consistent with the option name though. Keep in mind that this PR is to unify descriptions, so they all start with a capital letter and end with a dot, not to make them as authored by Shakespeare :wink: